assembl/static2/css/variables.scss

```diff
@@ -123,6 +123,7 @@ $font-size-sm: 16px;
 $font-size-m: 18px;
 $font-size-l: 26px;
 $font-size-xl: 36px;
+$font-size-xxl: 48px;
```

Comment:
We could even add these to styleguide, since they aren't there.

Reply:
the more things are in the styleguide, the happier I am !